cerberus/cerberus.py

```diff
@@ -327,6 +333,13 @@ def validate_schema(self, schema):
                             raise SchemaError(errors.ERROR_UNKNOWN_RULE % (
                                 constraint, field))
 
+    def _validate_coerce(self, coerce, field, value):
+        try:
+            value = coerce(value)
```

Comment:
The `coerce` argument is the callable that you define in the schema.  See https://github.com/nicolaiarocci/cerberus/pull/71/files#diff-aebc53cd4926f3a579adb7ba188de369R214